modules/sharedaddy/sharing.php

```diff
@@ -262,7 +267,7 @@ public function management_page() {
 						<h3><?php _e( 'Live Preview', 'jetpack' ); ?></h3>
 					</td>
 					<td class="services">
-						<h2<?php if ( count( $enabled['all'] ) > 0 ) echo ' style="display: none"'; ?>><?php _e( 'Sharing is off. Add services above to enable.', 'jetpack' ); ?></h2>
+						<h2<?php if ( count( $enabled['all'] ) > 0 ) { echo ' style="display: none"';} ?>><?php _e( 'Sharing is off. Add services above to enable.', 'jetpack' ); ?></h2>
```

Comment:
@zinigor i think multiple line condition are not work here, since it will looks little mess. The context of condition are inside tag parameter. it will looks like this

`<h2
<?php if ( count( $enabled['all'] ) > 0 ) {
 echo ' style="display: none"'; } 
?>><?php _e( 'Sharing is off. Add services above to enable.', 'jetpack' ); ?></h2>`

do you think it's ok?